src/interpreter/main.cpp

```diff
@@ -0,0 +1,12 @@
+#include <iostream>
+#include "../include/VM.h"
```

Comment:
You can use `#include <..>` here instead (in particular, you then avoid having to use relative paths).